code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -2478,3 +2478,11 @@
 	icon_state = "tatortot"
 	list_reagents = list("nutriment" = 4)
 	filling_color = "FFD700"
+
+/obj/item/weapon/reagent_containers/food/snacks/cucumberslice
+	name = "Cucumber Slice"
+	desc = "A slice of cucumber"
+	icon_state = "cucumberslice"
+	list_reagents = list("nutriment" = 1)
```

Comment:
Nutriment is typically a "typeless" form of nutrient found in processed foods.

Plantmatter is exactly what it sounds like: plant-derived nutrients.

This isn't me saying they aren't nutritious, this is me saying a carnivore shouldn't suddenly benefit nutritionally from them because you applied a knife. That would be like an herbivore suddenly being nutritionally satisfied by a raw cutlet because you sliced up the meat.

Protein for meat-based, plantmatter for plant-based, and nutriment for the "processed/cooked or otherwise unknown" foods.